Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM fedora:26
```

Comment:
is this for 1.7 support only?  We generally stay on centos and rhel images.

Reply:
@pweil- No. 1.8. centos don't have it.

Reply:
so we're bumping travis to 1.7 but bumping this to 1.8.  I'm confused as to why.

Reply:
`centos` was updated and now there is everything you need.
Fixed.